internal/state/actions.go

```diff
@@ -13,6 +13,10 @@ import (
 )
 
 func (c *state) WaitForActions(cmd *cobra.Command, ctx context.Context, actions ...*hcloud.Action) error {
+	if noWait, _ := cmd.Flags().GetBool("no-wait"); noWait {
+		return nil
```

Comment:
Should we log the action that we did not wait for? This way the user knows that something is still happening in the background and they could theoretically request the action status at a later time through the API.

Reply:
Sounds good :+1: